### DISCUSSION TITLE
Is `reportImplicitStringConcatenation` expected to ignore string literals with no commas inside parentheses?

### DISCUSSION THREAD

**Discussion by Azureblade3808** [microsoft/pyright#5800] [General]
I've found that when `reportImplicitStringConcatenation` is enabled, something like
```python
_ = [
    "ABC"
    "DEF"
]
```
is reported, but something like
```python
_ = (
    "ABC"
    "DEF"
)
```
is NOT reported.

Is this intended or a bug?

**Comment by erictraut** [comment]
This is intended. The check specifically exempts concatenated strings if surrounded by parens. See https://github.com/microsoft/pyright/issues/2781.

**Reply by Azureblade3808** [reply]
Thank you for the reply. 

I'm now wondering how I could have missed that post with so many common keywords.
